src/core/src/shape.cpp

```diff
@@ -9,12 +9,21 @@
 using namespace std;
 
 std::ostream& ov::operator<<(std::ostream& s, const Shape& shape) {
-    s << "{";
+    s << "[";
     s << ngraph::join(shape);
-    s << "}";
+    s << "]";
     return s;
 }
 
+size_t stringToSizeT(const std::string& valStr) {
```

Comment:
I get a warning
`In file included from /home/vzlobin/r/openvino/build/src/core/CMakeFiles/ngraph_obj.dir/Unity/unity_34_cxx.cxx:7:
/home/vzlobin/r/openvino/src/core/src/shape.cpp:18:8: warning: no previous declaration for ‘size_t stringToSizeT(const string&)’ [-Wmissing-declarations]
   18 | size_t stringToSizeT(const std::string& valStr) {`
on ubuntu. But I can't get why it mentions Unity. Can you take a look

Reply:
It's still there(

Reply:
I have two solutions: wrap the function into a nameless namespace
or remove the function. The last option will allow you to reuse `std::istringstream` you create inside of the function in a while loop.

BTW, you have `std` in `std::istringstream`, but you have `using namespace std;` on top

Reply:
Thanks, used nameless namespace and removed unnecessary std's.